erpnext/patches/v12_0/set_task_status.py

```diff
@@ -4,7 +4,7 @@ def execute():
     frappe.reload_doctype('Task')
 
     # add "Completed" if customized
-    for doctype in ('Task'):
+    for doctype in ['Task']:
```

Comment:
why for loop for a single item?

Reply:
I dont' know :man_shrugging: . I'll remove the loop.